### DISCUSSION TITLE
BBC World News Europe

### DISCUSSION THREAD

**Discussion by kheohoanganh** [iptv-org/community#960] [Show and tell]
### Channel ID

BBCWorldNewsEurope.uk

### Stream URL (optional)

http://152.167.4.130:8000/play/a00v/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://152.167.4.130:8000/play/a00v/index.m3u8 (offline)
<img width="350" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/8fc3edb8-ff0a-4fd4-a0fd-c0f36c9f6701">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/am_ucom.m3u#L17-L19
